python-backend/app/__init__.py

```diff
@@ -127,3 +122,14 @@ def default_error_handler(error):
                 'message': str(error)
             }
         }, getattr(error, 'code', 500)
+
+
+def register_scheduled_jobs(app):
```

Comment:
You don't have to refactor this as part of your PR but if we switch to using decorators instead of importing `_schedule_XXXX_jobs ` methods.
We can eliminate this whole function and every `_schedule_XXXX_jobs ` function in each scheduled taks

Reply:
I don't mind seeing it all in one place, but if it continues to grow this might be a good option.